### PR TITLE
HPCC-15803 Security Advance menu

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-15803 Document Advanced (Security) Menu
Document the Advanced (Security) Menu in ECLWatch.

@miguelvazq please review
@RichardTaylorHPCC please review
@JamesDeFabia  OPTIONALLY please review

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [ x ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ x ] My code follows the code style of this project.
  - [ x ] My code does not create any new warnings from compiler, build system, or lint.
- [ x ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ x ] My change requires a change to the documentation.
  - [ x ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ X ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:
Tested by building the doc set, with this change. 
Doc built properly.

### CODE REVIEW

File: docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml [hunk 1]
```diff
@@ -2107,4 +2114,62 @@
...
+
+    <para>Using the Advanced Menu with the Workunit Scopes selected opens only
+    the Default Permissions tab for the Workunit Scope (Default)
+    Permissions.</para>
+
+    <para><emphasis role="bold">NOTE:</emphasis> File or Workunit Scope
+    security needs to be enabled in your system configuration in order to use
+    File or Workunit Scope security on your system.</para>
+
+    <!-- /*Nother Note: Might be a good idea here to either include the doc for configuring your system to use File or Workunit scopes, or to specify how right here */
```
Comment (miguelvazq) [inline docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml]: Should this be Another?

Comment (g-pan) [inline docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml]: That is a non-printing /*comment*/ only displays in the source file,

Comment (richardkchapman) [inline docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml]: Still should be typo-free...

**Comment by g-pan** [comment]
Revised.

**Comment by richardkchapman** [comment]
Looks good - can you squash to a single commit please and I will merge

**Comment by g-pan** [comment]
Squashed.

**Pull request merged**